common/data/src/main/java/org/thingsboard/server/common/data/StringUtils.java

```diff
@@ -98,4 +97,51 @@ public static Iterable<String> split(String value, int maxPartSize) {
         return Splitter.fixedLength(maxPartSize).split(value);
     }
 
+    public static boolean equalsIgnoreCase(String str1, String str2) {
+        return str1 == null ? str2 == null : str1.equalsIgnoreCase(str2);
+    }
+
+    public static String join(String[] keyArray, String lwm2mSeparatorPath) {
+        return org.apache.commons.lang3.StringUtils.join(keyArray, lwm2mSeparatorPath);
+    }
+
+    public static String trimToNull(String toString) {
+        return org.apache.commons.lang3.StringUtils.trimToNull(toString);
+    }
+
+    public static boolean isNoneEmpty(String str) {
+        return org.apache.commons.lang3.StringUtils.isNoneEmpty(str);
+    }
+
+    public static boolean endsWith(String str, String suffix) {
+        return org.apache.commons.lang3.StringUtils.endsWith(str, suffix);
+    }
+
+    public static boolean hasLength(String str) {
+        return org.springframework.util.StringUtils.hasLength(str);
+    }
+
+    public static boolean isNoneBlank(String str) {
```

Comment:
Maybe ```isNotBlank```?

Reply:
@ashvayka should I create isNotBlank method and replace usages of isNoneBlank to isNotBlank where it is possible?
these usages were not written by me and I'm pretty sure that this replacement won't make any difference, because the only difference between isNoneBlank and isNotBlank is that the first one accepts CharSequence... instead of CharSequence and does the same job iteratively

Reply:
fixed